scripts/release/release_helpers.py

```
@@ -671,6 +673,131 @@ def command_artifact_manifest(args: argparse.Namespace) -> int:
+    """Load snapshot metadata XML from a local fixture or the live snapshot repository."""
+    if args.metadata_file:
+        source = args.metadata_file.resolve().as_uri()
+        return args.metadata_file.read_text(encoding="utf-8"), source
+
+    request = urllib.request.Request(
+        args.metadata_url,
+        headers={
+            "Accept": "application/xml",
+            "User-Agent": "ta4j-release-automation",
+        },
+    )
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)